.circleci/config.yml

```diff
@@ -274,10 +274,6 @@ jobs:
       - checkout
       - trust-github-key
       - update-carthage-integration-commit
-      # Carthage
-      - restore_cache:
-          keys: 
-            - carthage-cache-{{ checksum "Cartfile.resolved" }}
```

Comment:
this wasn't working, since `Cartfile.resolved` was removed [a while back](79008f567427791550fed31beaa441f2eef7f380) when we switched to SPM. 
This wasn't causing issues, though, since the job [would succeed](https://app.circleci.com/pipelines/github/RevenueCat/purchases-ios/3949/workflows/02a690c1-5941-4acc-a09e-895771fa80bc/jobs/10648) even if the step failed.